src/c4/C4_Serial.hh

```diff
@@ -73,6 +73,14 @@ DLL_PUBLIC_c4 int send_udt(const T * /*buffer*/, int /*size*/,
   return C4_SUCCESS;
 }
 
+template <typename TS, typename TR>
+DLL_PUBLIC_c4 int send_receive(TS * /*sendbuf*/, int /*sendcount*/,
+                               int /*destination*/, TR * /*recvbuf*/,
+                               int /*recvcount*/, int /*source*/,
+                               int /*sendtag*/, int /*recvtag*/) {
+  Insist(false, "send_receive is not support for C4_SCALAR builds.");
```

Comment:
I looked at this, and the problem is that there are separate tags for the send and receive halves. When they are identical, this is indeed a simple copy. Since I'm not sure how to handle this for tags that are not identical (I'm not sure it *can* be handled in serial) I'm disinclined to half-implement it. What do you think?